src/Tribe/Commerce/Currency.php

```diff
@@ -580,4 +580,256 @@ public function get_symbols_for_codes( $codes ) {
 	public function get_currency_code() {
 		return $this->currency_code;
 	}
+
+	/**
+	 * Get the Currency Decimal Point for a Provider.
+	 *
+	 * @since TBD
+	 *
+	 * @param string|null $provider The ticket provider class name.
+	 *
+	 * @return string The decimal separator.
+	 */
+	public function get_currency_decimal_point( $provider = null ) {
+
+		if ( ! class_exists( $provider ) ) {
+			return $this->get_currency_locale( 'decimal_point' );
+		}
+
+		if ( 'Tribe__Tickets_Plus__Commerce__WooCommerce__Main' === $provider ) {
+			$decimal_sep = get_option( 'woocommerce_price_decimal_sep' );
```

Comment:
We can just return here (and below) instead of setting a variable - unless you are going to filter it?

Reply:
```suggestion
			return get_option( 'woocommerce_price_decimal_sep' );
```